webrtc.html

```diff
@@ -655,33 +654,12 @@ <h4>Offer/Answer Options</h4>
         the offer/answer creation process.</p>
         <div>
           <pre class="idl"
->dictionary RTCOfferAnswerOptions {
-  boolean voiceActivityDetection = true;
-};</pre>
+>dictionary RTCOfferAnswerOptions {};</pre>
```

Comment:
Note: After this PR there is a single reference to voiceActivityDetection that still remains: The one inside of "dtx" attribute definition which is removed in #2351. Not touching it here avoids a merge conflict. But if that PR does not land this PR need to be updated to remove that reference.